clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -783,7 +784,13 @@ public void unsubscribe(TopicPartition... partitions) {
     private void scheduleAutoCommitTask(final long interval) {
         DelayedTask task = new DelayedTask() {
             public void run(long now) {
-                commit(CommitType.ASYNC);
+                commitAsync(new OffsetCommitCallback() {
```

Comment:
I thought the error message might be different because this one explicitly says "Auto commit".  Would User want to distinguish between auto commit and normal commit? If we don't care about the error message, we may simply call commitAsync() and the DefaultOffsetCommitCallback will be used.